extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/artnet-bridge.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- ArtNet Bridge -->
+    <bridge-type id="artnet-bridge">
+        <label>ArtNet Bridge</label>
+        <description>The ArtNet bridge represents a single DMX universe connected via ArtNet, only unicast</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
+                <label>Network Address</label>
+                <description>Network address of bridge, format: address[:port]. Default port is 6454.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="localaddress" type="text">
+                <label>Local Network Address</label>
+                <description>Network address of the sending host, format: address[:port]. Default port is 0 (random)</description>
+                <required>false</required>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="universe" type="integer" min="0" max="32767">
+                <label>DMX Universe</label>
+                <description>ID of DMX universe (0-32767)</description>
+                <default>0</default>
+                <required>false</required>
+            </parameter>
+            <parameter name="refreshmode" type="text">
+                <label>Refresh mode</label>
+                <description>Suppress re-transmission and refresh every 800ms or send every packet.</description>
+                <required>false</required>
+                <options>
+                    <option value="always">Always</option>
+                    <option value="standard">Standard</option>
+                </options>
+                <default>standard</default>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="applycurve" type="text">
+                <label>Apply Curve</label>
+                <description>List of channels that should use LED dim curve. Format is channel[,channel, ...] or channel[/width].</description>
```

Comment:
I think the order is guaranteed if multiple=true is used, but I am not so sure how well rearrangement of the order works through the Paper UI. I see that refactoring this would be a bigger effort without too much benefit for the users as it is anyhow a very technical setting/binding. So I'm ok with leaving it as it is.